steps/snyk/0.1.0/step.yml

```diff
@@ -0,0 +1,100 @@
+title: Snyk
+summary: |
+  Bitrise Snyk Step incorporates Snyk into your Bitrise workflows
+description: |
+  Bitrise Snyk Step incorporates Snyk into your Bitrise workflows.
+  By utilizing this step, you will be able to test your project dependencies for vulnerabilities during builds and use Snyk to monitor your projects
+  for new vulnerabilities.
+  This step uses Snyk as an external service and requires your authentication token.
+website: https://github.com/pilvikala/bitrise-step-snyk
+source_code_url: https://github.com/pilvikala/bitrise-step-snyk
+support_url: https://github.com/pilvikala/bitrise-step-snyk/issues
+published_at: 2022-08-18T08:55:31.184351151+03:00
+source:
+  git: https://github.com/pilvikala/bitrise-step-snyk.git
+  commit: 7403ce4ccc6883344be2922c8e095b2ddad75f8e
+project_type_tags:
+- android
+- react-native
+type_tags:
+- utility
+- security
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- auth_token: $SNYK_AUTH_TOKEN
+  opts:
+    description: |
+      Your Snyk authentication token (see https://app.snyk.io/account).
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Snyk Authentication Token
+    title: Auth Token
+- command: test
+  opts:
+    description: |
+      This is the CLI command to run with Snyk.
+
+      Default value: `test`
+    is_required: false
+    summary: Snyk CLI command to run
+    title: Command
+- opts:
+    description: |
+      Only report vulnerabilities of the provided level or higher (low/medium/high/critical).
+
+      Default is low.
+    summary: Only report vulnerabilities of the provided level or higher.
+    title: Severity threshold
+    value_options:
+    - low
+    - medium
+    - high
+    - critical
+  severity_threshold: low
+- fail_on_issues: "true"
+  opts:
+    description: |
+      Specifies whether to fail the build or not based on the results found by Snyk.
+
+      Snyk by default returns an error code from the test command. This may break your Bitrise workflow. By specifying `false`, the build can continue even if vulnerabilities are found.
+    summary: Specifies whether to fail the build or not based on the results found
+      by Snyk.
+    title: Fail on issues
+    value_options:
+    - "true"
+    - "false"
+- monitor: "false"
+  opts:
+    description: |
+      If enabled, imports the snapshot of dependencies to Snyk for continuous monitoring after a successful test.
+
+      Set this value to `true` to import the snapshot of dependencies to Snyk after a successful test. Snyk will then start monitoring the dependencies for new vulnerabilities and alert when a new vulnerability is discovered.
+    summary: If enabled, imports the snapshot of dependencies to Snyk for continuous
+      monitoring after a successful test.
+    title: Monitor (import to Snyk)
+    value_options:
+    - "false"
+    - "true"
+- opts:
+    description: ""
```

Comment:
Please add the description (at least the same text as the summary).

Reply:
will be addressed in a new PR